### ISSUE TITLE
Ipe 2.7.2 crashes when trying to open color window

### ISSUE THREAD

**Issue by ikostits** [otfried/ipe#192]
When the current color is not named/specified by a style sheet (e.g. after pressing on "pick properties" from an object with an unnamed color), pressing on the color box to open the Color window crashes the program. I tested it on Windows and on Mac.

**Comment by Willem3141** [comment]
Just ran into this. Here's a stack trace:

```
/usr/local/share/ipe/7.2.7/lua/actions.lua:178: bad argument #2 to 'find' (string expected, got table)
stack traceback:
        [C]: in method 'find'
        /usr/local/share/ipe/7.2.7/lua/actions.lua:178: in function </usr/local/share/ipe/7.2.7/lua/actions.lua:175>
        [C]: in function 'mainloop'
        /usr/local/share/ipe/7.2.7/lua/main.lua:467: in main chunk
        [C]: in function 'require'
        [string "require "main""]:1: in main chunk
```

The problem is that `self.attributes[button]` is a table like `{ "r" = 0.0, "g" = 0.0, "b" = 0.0 }` instead of a symbolic name, so then it shouldn't search for the color in the style sheet. I was able to fix this by replacing that part of the code by

```lua
    local old
    if type(self.attributes[button]) == "table" then
        old = self.attributes[button]
    else
        old = self.doc:sheets():find("color", self.attributes[button])
    end
    r, g, b = ipeui.getColor(self.ui:win(), "Select " .. button .. " color",
                old.r, old.g, old.b)
```

I'm not sure this is the cleanest solution, though.

**Comment by otfried** [comment]
Fixed in 7.2.8.
